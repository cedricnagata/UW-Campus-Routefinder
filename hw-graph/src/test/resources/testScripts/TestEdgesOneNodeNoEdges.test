# Create a graph
CreateGraph Graph

# Add a node
AddNode Graph n1

# List the children of n1 in the graph
ListChildren Graph n1