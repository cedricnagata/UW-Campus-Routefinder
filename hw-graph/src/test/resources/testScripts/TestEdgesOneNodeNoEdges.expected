# Create a graph
created graph Graph

# Add a node
added node n1 to Graph

# List the children of n1 in the graph
the children of n1 in Graph are: